app/queries/get_link_changes.rb

```diff
@@ -0,0 +1,83 @@
+module Queries
+  class GetLinkChanges
+    # Current maximum number of results. This is an arbitrary number to prevent
+    # too much data being returned, and to hold off from implementing pagination.
+    MAXIMUM_NUMBER_OF_RESULTS = 250
+
+    attr_reader :params
+
+    def initialize(params)
+      @params = params
+    end
+
+    def as_hash
+      results = link_changes.map do |link_change|
+        {
+          id: link_change.id,
```

Comment:
Is this field made use of it? Exposing it makes me feel like you should be able to do something with it - like look up the change directly, which we don't allow, so just wanted to check that we expose this for a reason and it's used.

Reply:
Removed.